lms/templates/body-initial.html

```diff
@@ -1,4 +1,15 @@
+<%namespace file='/theme-variables.html' import="get_global_settings" />
 <%!
   from hijack.templatetags.hijack_tags import _render_hijack_notification
 %>
 ${_render_hijack_notification((request))}
+
+% if (get_global_settings().get('customer_gtm_enabled') and get_global_settings().get('customer_gtm_id')):
+  <%
+    customerGtmId = get_global_settings().get('customer_gtm_id')
```

Comment:
Unless we're doing sanity checking on the entry side, this could be hacked. Of course, we need to trust our customers to some extent, but input checking is always a good habit.

Reply:
Thanks @abeals for pointing this out. I've just commited the suggested changes @OmarIthawi made, so can we get the seal of approval now? :)

Reply:
Eh, I suppose. 😉